README.md

```diff
@@ -65,6 +65,16 @@ We will refer below to two types of capitalization:
 
 Use title format for all section, subsection, etc. titles. In order to help you capitalize the right words, there's a handy website: [capitalizemytitle.com](https://capitalizemytitle.com).
 
+## Maintaining name of the objects and its reference on the same line
+
+Sometimes, the name of an object (such as Figure, Table, Graph, or Algorithm) and its reference number are split into two lines. For instance, the name of the object may be on one line, while the reference number appears on the next line. To ensure that LaTeX keeps both the name of the object and its reference on the same line, you can use the character "~" between the object and the reference.
```

Comment:
nit: as suggested in the doc, can you please have one sentence per line :) ?

Reply:
Of course!